docs/02-app/02-api-reference/01-directives/use-cache.mdx

```diff
@@ -239,6 +239,7 @@ For example, if you add the `use cache` directive to your page, without specifyi
 ```tsx filename="app/components/parent.tsx" highlight={5,6,19,20}
 // Parent component
 import { unstable_cacheLife as cacheLife } from 'next/cache'
+import { ChildComponent } from './child'
```

Comment:
We don't need to import it since it's in the same file. But, thinking about it, maybe we should make it more clearer that these can be in different folders by having two code blocks, then Parent can import the Child component. 

Would you like to make that change @Developerayo?

Reply:
For sure! Pushed...

Reply:
Thank you 🫶